healthchecks/src/model.rs

```diff
@@ -203,6 +203,32 @@ impl Default for UpdatedCheck {
     }
 }
 
+/// Represents a list of pings this check has received.
+#[derive(serde::Deserialize)]
+pub struct Pings {
+    pub pings: Vec<Ping>,
+}
```

Comment:
This needs to be private in the method that needs it, and the method should instead expose a `anyhow::Result<Vec<Ping>>`. https://github.com/msfjarvis/healthchecks-rs/commit/977d63b73385f64b6cd6bc23343d9421e2ce19ed for reference.

Reply:
Hi @msfjarvis, PR updated!